api.raml

```diff
@@ -26,6 +26,18 @@ securitySchemes:
 # Data Service Specific Endpoints
 ####################################
 
+/filter-aware-metadata/continuous-variable:
```

Comment:
Side note, we don't actually even have to expose this as an endpoint if we don't want to (would still need to be wired up in the service class anyway), since it can be called directly by the dependent plugins.  The only thing we actually lose is testability.  Not saying we shouldn't expose, just that we don't have to.

Reply:
Ok, sounds good.  I'm fine with this endpoint name.